pkg/controller/command/vcwallet/command.go

```diff
@@ -888,7 +909,14 @@ func (o *Command) ProposeCredential(rw io.Writer, req io.Reader) command.Error {
 		return command.NewExecuteError(ProposeCredentialErrorCode, err)
 	}
 
-	msg, err := vcWallet.ProposeCredential(request.Auth, request.Invitation,
+	didComm, err := wallet.NewDidComm(vcWallet, o.ctx)
```

Comment:
__follow up task__: can you check if `wallet.NewDidComm()` is expensive in terms of store initialization, kms operaton etc? if yes then there will be performance implications initializing this instance everytime inside this wallet function.
Work around can be `client.NewWallet() //for regular wallet & client.NewDIDCommWallet() // for didcomm wallet`.

Reply:
Only NewLookup service inside wallet.NewDidComm is accessing storage. It just calls SetStoreConfig, this shouldn't be expensive. And no KMS operations are called in NewDidComm.